packages/js/components/src/form/stories/index.js

```diff
@@ -68,6 +73,14 @@ export const Basic = () => {
 								] }
 								{ ...getInputProps( 'select' ) }
 							/>
+							<DateTimePickerControl
+								label="Date"
+								dateTimeFormat="YYYY-MM-DD HH:mm"
+								placeholder="Enter a date"
+								{ ...getInputProps( 'date', {
+									alias: { value: 'currentDate' },
```

Comment:
Should the `alias` be switched around here -> `{ currentDate: 'value' }`?
The way it reads now, my thought is that it would be adding a `value` prop to the DateTimePickerControl pointing to `currentDate`, but from my understanding of the logic, it's the other way right?

As I side note, I don't mind this addition, but was curious how beneficial it would be, is it not easier to just get all the values from the context and add the `currentDate={ values['data'] }` prop?

Reply:
@louwie17 Stop pushing back against the addition of unnecessary complexity. Seriously though, you are correct... this whole `alias` concept just isn't needed. At least not for the specific scenario that I'm trying to solve with `DateTimePickerControl`. I'm going to pull this out. I think all that will remain from the PR will be Storybook story addition and the unit test.

Thanks for continuing to push back on this... the resulting API is better for it.